build.gradle

```diff
@@ -20,8 +20,8 @@ ext {
     automatticRestVersion = '1.0.8'
     automatticStoriesVersion = '2.4.0'
     automatticTracksVersion = '3.3.0'
-    gutenbergMobileVersion = 'v1.109.1'
-    wordPressAztecVersion = 'v1.8.0'
+    gutenbergMobileVersion = '6435-27269ff224367b0e280b58e74247ba50e74ec32d'
+    wordPressAztecVersion = '1070-e05c8afb0960a2419c3a5fd13204ea4c35d0101f'
```

Comment:
@SiobhyB This reference points to a testing PR that is currently closed (https://github.com/wordpress-mobile/AztecEditor-Android/pull/1070#top). Not sure if we should update it and whether it would affect the installable build.

Reply:
@fluiddot, ah, thanks for pointing that out! Updated to the correct reference in https://github.com/wordpress-mobile/WordPress-Android/pull/19724/commits/d81494893bceadb8c829e4f6728a97d23abbdf27.